### PR TITLE
feat: add new works from galleries you follow

### PR DESCRIPTION
**Pull request by MounirDhahri**
This PR resolves [ONYX-1175]

This PR adds new works from galleries you follow artworks to the home view.

Screenshot 2024-08-07 at 12 54 07

[ONYX-1175]: https://artsyproduct.atlassian.net/browse/ONYX-1175?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/schema/v2/homeView/sections.ts [hunk 1]
```diff
@@ -53,9 +63,10 @@ export const TrendingArtists: HomeViewSection = {
 }
 
 const sections: HomeViewSection[] = [
-  RecentlyViewedArtworks,
   AuctionLotsForYou,
   NewWorksForYou,
+  NewWorksFromGalleriesYouFollow,
+  RecentlyViewedArtworks,
```
Comment (olerichter00) [inline src/schema/v2/homeView/sections.ts:69]: 👀

**Review by olerichter00 — APPROVED** [review]
👏

**Comment by olerichter00** [comment]
Can we add tests for the section?

File: src/schema/v2/homeView/sections.ts (hunk 1 above)
Comment (MounirDhahri) [reply src/schema/v2/homeView/sections.ts:69]: I just ordered this alphabetically  - recently viewed was the first rail

**Review by nickskalkin — DISMISSED** [review]
👍

**Pull request merged**